D/DynamicalBilliards/Compat.toml

```diff
@@ -22,5 +22,8 @@ julia = "0.7-1"
 ["2.4"]
 Elliptic = "0.5-0"
 
-[3]
+["3-3.5.1"]
 Elliptic = "0.5-0"
```

Comment:
and this?

Reply:
I don't know :( sorry. I don't know what the Compat.toml is supposed to contain.

What I can tell you for sure is that it was **not** my intention to change any versioning bounds while changing to the Registrator/Project.toml setting.

Reply:
Well, then add compat for these deps too? As you had in your REQUIRE